Barotrauma/BarotraumaShared/SharedSource/Characters/Jobs/JobPrefab.cs

```diff
@@ -192,6 +193,14 @@ public bool HiddenJob
             private set;
         }
 
+        //whether the job should be only available to players
+        [Serialize(false, false)]
+        public bool PlayerOnly
```

Comment:
Perhaps this could be merged with `HiddenJob` using an enum?
```cs
[Flags]
public enum Availability {
    Player = 0x1,
    Npc = 0x2,
    Both = 0x3
}

[Serialize(Availability.Both, false)]
public Availability AvailableTo;```